### DISCUSSION TITLE
Beginner Friendly

### DISCUSSION THREAD

**Discussion by armeetj** [monkeytypegame/monkeytype#1693] [Questions]
Perhaps a STRUCTURE.md file could be added or, a wiki to explain how the code is structure/works?
There are thousand line files scattered all over the place. This document could help developers understand what they're looking at.
I am new to the "full-stack" web dev scene, so maybe its just me. The most I've ever done is HTML/CSS and basic JS frameworks like Vue/react, so I don't have a good understanding of how this website works.

**Comment by tajhans** [comment]
Information regarding the structure and file types is in the [Contributing Guidelines](https://github.com/Miodec/monkeytype/blob/c8f2db689aabc19eb0496e2d5167d2c4afe0f4a9/CONTRIBUTING.md) I think at least.

Also if you find it hard to navigate through files click the <kbd>go to file</kbd> button. It makes it easier to navigate through files.

**Reply by unrealapex** [reply]
> Perhaps a STRUCTURE.md file could be added or, a wiki to explain how the code is structure/works?

Proposed in #1334, but didn't get a response :(.
Most of the info you will need is in the [CONTRIBUTING.md](https://github.com/Miodec/monkeytype/blob/c8f2db689aabc19eb0496e2d5167d2c4afe0f4a9/CONTRIBUTING.md).

**Reply by unrealapex** [reply]
> Also if you find it hard to navigate through files click the go to file button. It makes it easier to navigate through files.

Didn't know you could do that, good tip 👍 .

**Reply by armeetj** [reply]
@teddinotteddy The Contributing guidelines aren't helpful in explaining what parts of the project do what, the process flow of the project, etc.
